mattstauffer/tuber
# Install and run Duster, require and install Breeze, add YouTube videos to the home page

TL;DR: Sets up the basic tooling required for the site.

---

Installs and runs Duster. Closes #1.
Todo talk about duster
Todo talk about breeze

YouTube Videos:
Why are we doing this? Sally requested () to do (). 

To test:
1. Get a YouTube API key from (link here)[https://www.google.com] with x and y permissions
2. Add an environment variable named `YOUTUBE_API_KEY` to your `.env` file with that API key
3. Open up the home page in your browser and make sure you see a list of videos from Matt's YouTube channel

<img width="837" alt="image" src="https://user-images.githubusercontent.com/151829/216717282-fdb5b282-3834-449d-93e7-6b8c79b5dd4a.png">
